build.sbt

```diff
@@ -151,17 +151,8 @@ onLoad.in(Global) ~= { old =>
 cancelable.in(Global) := true
 crossScalaVersions := Nil
 
-lazy val execScalafmt = taskKey[Unit]("Execute scalafmt executable in /bin")
-execScalafmt := {
-  "bin/scalafmt" !
-}
-
 addCommandAlias("scalafixAll", "all compile:scalafix test:scalafix")
 addCommandAlias("scalafixCheck", "; scalafix --check ; test:scalafix --check")
-addCommandAlias(
-  "preparePr",
-  "scalafixAll ; execScalafmt"
-)
```

Comment:
Not related to fastpass, but I also wanted to remove this. It wasn't a great idea since after running this scalafmt is still ran in the hook. So you needlessy run scalafmt twice. Good idea, bad execution. I still think it'd be nice to have something like this, but we'd need to think about it a bit more.